Source/Core/DolphinQt/DolphinQt.vcxproj

```diff
@@ -403,7 +404,7 @@
       <Project>{0e033be3-2e08-428e-9ae9-bc673efa12b5}</Project>
     </ProjectReference>
     <!--
-      This project doesn't uses its own PCH during compile (because RTTI setting differs),
+      This project uses its own PCH during compile (because RTTI setting differs),
       but we must also link the "main" pch for the dependants (DolphinLib)
     -->
     <ProjectReference Include="$(SourceDir)PCH\pch.vcxproj">
```

Comment:
I assume this was a typo in #10772?

Reply:
initially when moving to Qt6 i disabled pch on DolphinQt and added the original comment. Then I enabled pch for DolphinQt and modified the comment, and made a typo.